src/components/datatables/Actions.tsx

```diff
@@ -70,6 +92,10 @@ export function Actions(props: Props) {
     }),
   };
 
+  const customFiltersPlaceholders: Placeholders = {
+    client_status: t('invoice_status'),
```

Comment:
I assume this will be fixed once interface is changed, so make sure to have this covered as well.